docs/user/07-70-function-specification.md

```diff
@@ -0,0 +1,254 @@
+| **buildResponseCloudEvent** | id, type, data | **Deprecated: use `emitCloudEvent` instead.** Builds a CloudEvent object based on the `request` CloudEvent object and the given arguments |
+| **emitCloudEvent** | type, source, data, optionalCloudEventAttribute | Builds a CloudEvent based on the arguments and emits it on the eventing publisher service. You can pass any additional [cloudevent attributes](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/json-format.md#2-attributes) as properties of the last optional argument `optionalCloudEventAttribute` |
+
+</details>
+<details>
+<summary label="Python">
+Python
+</summary>
+
+| Method name | Arguments | Description |
+|----------|-----------|-------------|
+| **publishCloudEvent** | event | Publishes a CloudEvent on the publisher service based on the given CloudEvent object |
+| **buildResponseCloudEvent** | id, type, data | Builds a CloudEvent object based on the `request` CloudEvent object and the given arguments |
+
+</details>
+</div>
+
+### Context object
+
+The `context` object contains information about the Function's invocation, such as runtime details, execution timeout, or memory limits.
+
+See sample context details:
+
+```json
+...
+{ "function-name": "main",
+  "timeout": 180,
+  "runtime": "nodejs18",
+  "memory-limit": 200Mi }
+```
+
+See the detailed descriptions of these fields:
+
+| Field | Description                                                                               |
+|-------|-------------------------------------------------------------------------------------------|
+| **function-name** | Name of the invoked Function                                                              |
+| **timeout** | Time, in seconds, after which the system cancels the request to invoke the Function       |
+| **runtime** | Environment used to run the Function. You can use `nodejs16` - deprecated, `nodejs18`, or `python39`. |
+| **memory-limit** | Maximum amount of memory assigned to run a Function                                       |
+
+## HTTP requests
+
+You can use the **event.extensions.request** object to access properties and methods of a given request that vary depending on the runtime. For more information, read the API documentation for [Node.js Express](http://expressjs.com/en/api.html#req) and [Python](https://bottlepy.org/docs/dev/api.html#the-request-object).
+
+## Custom HTTP responses
+
+<div tabs name="custom-http-response" group="function-specification">
+<details>
+<summary label="Node.js">
+Node.js
+</summary>
+
+By default, a failing Function simply throws an error to tell the Event Service to reinject the event at a later point. Such an HTTP-based Function returns the HTTP status code `500`. On the contrary, if you manage to invoke a Function successfully, the system returns the default HTTP status code `200`.
```

Comment:
As this paragraph is identical for node.js and Python, you can extract it from the divs and use it as introductory paragraph above them.

```suggestion
```